app/models/season/phase_switcher.rb

```diff
@@ -1,9 +1,20 @@
+# frozen_string_literal: true
+
 class Season::PhaseSwitcher
 
   def self.season
     Season.current
   end
 
+  def self.destined(phase)
+    if phase.in?(whitelisted_phases)
```

Comment:
The silent failing is intentional, to diss mailicious input-ters. Do I miss a use case where it would be helpful? Can you think of one? See https://github.com/rails-girls-summer-of-code/rgsoc-teams/pull/585#discussion_r95774916